apps/gatsby/src/index.spec.jsx

```diff
@@ -79,12 +78,14 @@ describe('Gatsby Preview entry point', () => {
     fetchSpy.mockRestore();
     getElementByIdSpy.mockRestore();
     render.mockClear();
-    jest.unmock('@contentful/app-sdk');
+    vi.unmock('@contentful/app-sdk');
   });
 
-  it('should initialize the app', () => {
-    loadEntryPoint();
+  it('should initialize the app', async () => {
+    await import('./index');
+    // loadEntryPoint();
```

Comment:
Why is this commented out now? Do we not need to test it anymore?

Reply:
that needs to be uncommented, will fix